src/UID2.Client/UID2Encryption.cs

```diff
@@ -25,19 +25,20 @@ internal static DecryptionResponse Decrypt(string token, KeyContainer keys, Date
             string headerStr = token.Substring(0, 4);
             Boolean isBase64UrlEncoding = headerStr.IndexOfAny(BASE64_URL_SPECIAL_CHARS) != -1;
             byte[] data = isBase64UrlEncoding ? UID2Base64UrlCoder.Decode(headerStr) : Convert.FromBase64String(headerStr);
+            IdentityType? identityType = GetIdentityType((byte) ((data[0] & 12) >> 2));
```

Comment:
where do the &12 and >>2 come from, is this from a spec? Can we reference it and/or use local consts?